### PR TITLE
Update NuGet package metadata

### PR DESCRIPTION
**Pull request by adityapatwardhan**
## PR Summary

Update the nuget package metadata as per nuget.org requirements.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: tools/packaging/packaging.strings.psd1 [hunk 1]
```diff
@@ -109,22 +109,27 @@ OsxDistributionTemplate = @'
...
-    </dependencies>
-  </metadata>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+    <metadata>
+        <id>{0}</id>
+        <version>{1}</version>
+        <authors>Microsoft</authors>
+        <owners>Microsoft,PowerShell</owners>
+        <requireLicenseAcceptance>true</requireLicenseAcceptance>
+        <description>Native binaries for PowerShell Core</description>
```
Comment (TravisEz13) [inline tools/packaging/packaging.strings.psd1]: update description

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comments

**Comment by adityapatwardhan** [comment]
@TravisEz13  Fixed, please have a look.

**Review by TravisEz13 — APPROVED** [review]

File: tools/packaging/packaging.strings.psd1 [hunk 2]
```diff
@@ -109,22 +109,27 @@ OsxDistributionTemplate = @'
...
+        <authors>Microsoft</authors>
+        <owners>Microsoft,PowerShell</owners>
+        <requireLicenseAcceptance>true</requireLicenseAcceptance>
+        <description>PowerShell runtime for hosting PowerShell Core</description>
+        <projectUrl>https://github.com/PowerShell/PowerShell</projectUrl>
+        <iconUrl>https://github.com/PowerShell/PowerShell/blob/master/assets/Powershell_black_64.png?raw=true</iconUrl>
+        <licenseUrl>https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt</licenseUrl>
+        <tags>PowerShell</tags>
+        <language>en-US</language>
+        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
```
Comment (iSazonov) [inline tools/packaging/packaging.strings.psd1]: Is it important to use `©` symbol?

Comment (adityapatwardhan) [reply tools/packaging/packaging.strings.psd1]: Yes it is a requirement by NuGet.org

File: tools/packaging/packaging.strings.psd1 [hunk 3]
```diff
@@ -109,22 +109,27 @@ OsxDistributionTemplate = @'
 '@
 NuspecTemplate = @'
 <?xml version="1.0" encoding="utf-8"?>
-<package
-  xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
-  <metadata>
-    <id>{0}</id>
-    <version>{1}</version>
-    <title>PowerShellRuntime</title>
```
Comment (dantraMSFT) [inline tools/packaging/packaging.strings.psd1]: Did you intent to remove the <title> element?

**Review by dantraMSFT — APPROVED** [review]
Other than the question about the removal of <title> element, LGTM.

File: tools/packaging/packaging.strings.psd1 (hunk 3 above)
Comment (adityapatwardhan) [reply tools/packaging/packaging.strings.psd1]: According to nuget.org documentation having a description is a must, title is not mandatory. Hence, removed it.

**Comment by adityapatwardhan** [comment]
@TravisEz13 Ready to merge?

**Pull request merged**